swagger.json

```diff
@@ -0,0 +1,20 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "BAREFOOT NOMAD",
+    "description": "Making company tavel and accomodation easy and convinient"
```

Comment:
Hi @okafor-chidimma , please update the spellings in the description text. Some words are not spelled properly.

Reply:
@Stylll Please review, I have implemented the changes

Reply:
@okafor-chidimma I can't see the changes implemented.

Reply:
@Stylll Please check the last commit. I had to push again